fetch.bs

```diff
@@ -1800,6 +1805,15 @@ initially unset.
 being provided to an API that didn't make a range request. See the flag's usage for a detailed
 description of the attack.
 
+<p>A <a for=/>response</a> has an associated
+<dfn for=response id=concept-response-timing-allow-failed-flag>timing allow check flag</dfn>, which
```

Comment:
The id and the name don't match. It's "failed" in one and "check" in the other

Reply:
Would be better to make the name more explicit. IIUC, this flag is set when TAO check passed successfully. So maybe "timing allow passed flag"?

Reply:
Done